docs/jumpcloud.md

```diff
@@ -0,0 +1,21 @@
+# Integrating the JumpCloud Provider
+
+## Example
+
+```csharp
+services.AddAuthentication(options => /* Auth configuration */)
+        .AddJumpCloud(options =>
+        {
+            options.ClientId = "my-client-id";
+            options.ClientSecret = "my-client-secret";
+            options.Domain = "https://oauth.id.jumpcloud.com";
+        });
+```
+
+## Required Additional Settings
+
+_None._
+
+## Optional Settings
+
+_None._
```

Comment:
`Domain` appears to be a new value (though not clear if required or not), so needs documenting.

Reply:
I have added the Domain details to the docs